mergin/client.py

```diff
@@ -482,5 +468,43 @@ def backup_if_conflict(path, checksum):
             move_file(local_path(file["path"]), local_path(file["new_path"]))
 
         local_info["files"] = server_info["files"]
-        local_info["version"] = server_info["version"]
+        local_info["version"] = server_info["version"] if len(server_info["version"]) else 'v0'
         save_project_file(directory, local_info)
+
+    def _download_file(self, project_path, project_version, file, directory):
+        """
+        Helper to download single project file from server in chunks.
+
+        :param project_path: Project's full name (<namespace>/<name>)
+        :type project_path: String
+        :param project_version: Version of the project (v<n>)
+        :type project_version: String
+        :param file: File metadata item from Project['files']
+        :type file: dict
+        :param directory: Project's directory
+        :type directory: String
+        """
+        chunk_size = 10 * 1024 * 1024
```

Comment:
Id rather define a chunk size out of the function body, maybe could be a parameter for a client in the future? (not sure how is this compatible with server, maybe will get chunk size from a server)

Reply:
in server it can be set up as env var, we can implement its return value, so client can in future match to it, for time being, as suggested, I would leave it as global const in client